src/components/index.js

```diff
@@ -0,0 +1,5 @@
+export {default as Graphic} from './graphic'
```

Comment:
Is this the same as `export Graphic from './graphic'`?

Reply:
I don't think that syntax works. Also I'm exporting all the components like that from components/index.js so they can easily be imported like this: import { Graphic, Hero, Section, Tagline } from "../components/";

Reply:
Never mind, I read this wrong.